cvat/apps/quality_control/serializers.py

```diff
@@ -38,6 +38,7 @@ class QualityReportSummarySerializer(serializers.Serializer):
     valid_count = serializers.IntegerField(source="annotations.valid_count")
     ds_count = serializers.IntegerField(source="annotations.ds_count")
     gt_count = serializers.IntegerField(source="annotations.gt_count")
+    total_count = serializers.IntegerField(source="annotations.total_count")
```

Comment:
I'm ok with the changes in the PR, but as we changed API, it required to update the test assets. Please run `(cd tests && python python/shared/utils/dump_objects.py)` so that `quality_reports.json` was updated.

Reply:
OK, done.